testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/MultiprocessOrchestration.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+using Debug = UnityEngine.Debug;
+
+public class MultiprocessOrchestration
+{
+    public const string buildInfoFileName = "buildInfo.txt";
+    public const string isWorkerArg = "-isWorker";
+
+    public static void StartWorkerNode()
+    {
+        var workerNode = new Process();
+
+        //TODO this should be replaced eventually by proper orchestration for all supported platforms
```

Comment:
Nit, but as a less-informed reader, this comment made me more confused than I was before. I'd suggest either nixing the comment or adding more comprehensive details